dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/servers/servlet40_sameSiteSecurity/server.xml

```diff
@@ -12,32 +12,40 @@
  -->
 <server description="Server for testing the Cookie SameSite attribute">
 
-    <include location="../fatTestPorts.xml"/>
+    <include location="../fatTestPorts.xml" />
 
     <featureManager>
         <feature>servlet-4.0</feature>
         <feature>appSecurity-2.0</feature>
         <feature>jsp-2.3</feature>
     </featureManager>
 
-    <keyStore id="defaultKeyStore" password="{xor}EzY9Oi0rJg=="/> <!-- pwd: Liberty, expires 1/4/2099 -->
+    <httpEndpoint id="defaultHttpEndpoint"
+        host="*"
+        httpPort="${bvt.prop.HTTP_default}"
+        httpsPort="${bvt.prop.HTTP_default.secure}">
+        <samesite lax="*" />
```

Comment:
Server.xml was different fro the javadoc: 

https://github.com/OpenLiberty/open-liberty/blob/16197b274fe83d51a34062c47cea00e1bc4e80a4/dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCSameSiteCookieAttributeSecurityTest.java#L79C1-L81C54